src/Extension/Resources/config/task/AcpAppAsyncApiAddMessageSchemaTask.yaml

```diff
@@ -5,15 +5,15 @@ help: ~
 stage: build
 version: 0.2.0
 command: "php %executable_command% schema:asyncapi:message:add %channel-name% %message-name% %operation-id% %asyncapi-file% %property% %from-transfer-class% %message-type% %add-metadata% -v"
-type: local_cli_interactive
+type: !php/const SprykerSdk\SdkContracts\Enum\Task::TYPE_LOCAL_CLI_INTERACTIVE
```

Comment:
Task and task set versions must be increased respectively. Required for all the tasks and task sets.

Reply:
I updated but for DB nothing were changed